src/camera.rs

```diff
@@ -586,6 +586,23 @@ impl Camera {
         /// Read the sensor clock frequency in Hz
         sensor_clock_freq_hz: f32;
 
+        /// Data move policy
+        mut buffer_policy: i32;
+
+        /// Auto white balance mode.
+        mut auto_wb: i32;
```

Comment:
```suggestion
        mut auto_wb: XI_SWITCH::Type;
```
The parameter is a simple on/off switch, so it should have the appropriate type for that.